Bio/UniGene/__init__.py

```diff
@@ -250,8 +250,12 @@ def __init__(self):
 
     def __repr__(self):
         """Represent the UniGene Record object as a string for debugging."""
-        return "<%s> %s %s\n%s" % (self.__class__.__name__,
-                                   self.ID, self.symbol, self.title)
+        return "<%s> %s %s\n%s" % (
```

Comment:
This isn't a very standard ``__repr__`` string, the trailing ``\n`` is most unusual. Do you want to look at updating this since you are working on this module?

Reply:
I have removed the newline, as formatting should be part of __str__, not in scope of this change.